.circleci/config.yml

```diff
@@ -74,7 +74,7 @@ jobs:
   build-macos:
     macos:
       xcode: "14.1.0"
-    resource_class: macos.x86.medium.gen2
+    resource_class: macos.m1.medium.gen1
```

Comment:
:+1: had planned to do this but they new resources weren't available at the time